index.html

```diff
@@ -75,7 +75,7 @@
         ctx     = canvas.getContext('2d'),
         ucanvas = get('upcoming'),
         uctx    = ucanvas.getContext('2d'),
-        speed   = { start: 0.6, decrement: 0.005, min: 0.1 }, // how long before piece drops by 1 row (seconds)
+        speed   = { start: 0.9, decrement: 0.005, min: 0.6 }, // how long before piece drops by 1 row (seconds)
```

Comment:
```suggestion
        speed   = { start: 0.9, decrement: 0.005, min: 0.7 }, // how long before piece drops by 1 row (seconds)
```

I think `0.6` is too fast, how about `0.7`?